components/movementsensor/gpsrtkserial/gpsrtkserial_test.go

```diff
@@ -101,8 +101,15 @@ func TestReconfigure(t *testing.T) {
 	test.That(t, g.wbaud, test.ShouldEqual, 115200)
 }
 
+// initializePosition sets the position in the cached data and returns the point it is set to.
+func initializePosition(cachedData *gpsutils.CachedData) *geo.Point {
+	// This sets the position to 12°34.5678' N, 123°45.6789' W, at time 12:34:56.78 UTC
+	setPositionSentence := "$GPGLL,1234.5678,N,12345.6789,W,123456.78,A,D*7F"
```

Comment:
Make this sentence a variable

Reply:
Whoops! I totally forgot about this PR. We discussed offline: I've moved this from a variable to a global constant.